nucypher/blockchain/eth/actors.py

```diff
@@ -89,7 +86,7 @@
 from nucypher.network.nicknames import nickname_from_seed
 
 
-class NucypherTokenActor:
+class BaseActor:
```

Comment:
Interesting - Does this suggest that this base class is also used by actors that do not use perform token transactions?

Reply:
Yes. For example, the MultiSig is independent from the NuCypher token. It can use it, but it doesn't have to. That's actually the motivation for this.